src/main/java/org/reduct/client/ReductClient.java

```diff
@@ -54,4 +58,110 @@ <T> HttpResponse<String> sendRequest(String url, Method method) throws ReductExc
          throw new ReductSDKException("Thread has been interrupted while processing the request", e);
       }
    }
+
+   public Bucket getBucket(String name) throws ReductException, ReductSDKException, IllegalArgumentException {
+      return new Bucket(name, this).read();
+   }
+
+   public Bucket createBucket(String name) throws ReductException, ReductSDKException, IllegalArgumentException {
```

Comment:
Let's pass settings during the creation of the bucket:

1.  We need BucketSettings class for serialization
2. Add Bukcet.setSettings and Bucket.getSettings methods to mange the settings.

Reply:
Completed